OrcTests/test_data/performance/threads.orc

```diff
@@ -2,7 +2,7 @@
 This program creates 2^20 (or about 1 million) threads
 and waits for them to terminate.
 -}
-val N = 13
+val N = 18
```

Comment:
No. 2^20 takes to long on the old thread scheduler to be practical for benchmarking. I think we can play with it later and change it again.